mcp_server/src/mcp_server_neo4j_gds/gds.py

```diff
@@ -193,10 +207,12 @@ def get_relationship_properties_keys(gds: GraphDataScience, relationshipTypes=No
 def get_relationship_types(gds: GraphDataScience, node_labels=None):
     if node_labels is None:
         node_labels = []
-    query = (
-        create_relationship_cypher_match_query(node_labels, [])
-        + " RETURN DISTINCT TYPE(r) as relationship_types"
-    )
+    type_extractor = """
```

Comment:
Does the this type_extractor part need to be changed? I thought `RETURN DISTINCT TYPE(r) as relationship_types` worked.